src/main/scala/com/github/pjfanning/sourcedist/SourceDistGenerate.scala

```diff
@@ -23,6 +23,9 @@ private[sourcedist] object SourceDistGenerate {
     customIgnorePatterns.add(".git/")
     customIgnorePatterns.add(".github/")
     customIgnorePatterns.add(".git*")
+    customIgnorePatterns.add(".bsp/")
```

Comment:
Can you add a reference to https://github.com/pjfanning/sbt-source-dist/issues/14 here?